debian8/guide.xslt

```diff
@@ -34,9 +34,9 @@
 
       <xsl:apply-templates select="document(concat($BUILD_RP, '/bash-remediation-functions.xml'))" />
 
-      <xsl:apply-templates select="document(concat($SHARED_RP, '/xccdf/intro/shared_intro_os.xml'))" />
```

Comment:
@yuumasato this is the only thing to not change.

Reply:
@redhatrises Can you explain me why?

Reply:
We need the XCCDF that contains the shorthand groups which contain the rules. The shared intro has been migrated for all platforms and isn’t the issue that we are coming across in the yaml migration.

Reply:
+1, at this point don't move the shared intro, I will either deal with it in the yaml dumper script or I will copy it later.

Reply:
Okay, updated.